### PR TITLE
More bcl tests

### PR DESCRIPTION
**Pull request by c00crane**
Add more tests for back channel logout

### CODE REVIEW

**Comment by c00crane** [comment]
#build
#spawn.fullfat.buckets=com.ibm.ws.security.fat.common.backchannelLogout,com.ibm.ws.security.oidc.client_fat.backchannelLogout,com.ibm.ws.security.oidc.server_fat.backchannelLogout,com.ibm.ws.security.social_fat.LibertyOP.backchannelLogout

File: dev/com.ibm.ws.security.fat.common.backchannelLogout/fat/src/com/ibm/ws/security/backchannelLogout/fat/utils/AfterLogoutStates.java [hunk 1]
```diff
@@ -64,6 +64,8 @@ public class AfterLogoutStates {
     boolean isUsingEndSessionWithHttpLogout = false;
     boolean isUsingLogoutWithHttpLogout = false;
     boolean isUsingJwtToken = true;
+    boolean isUsingIntrospect = false;
+    boolean isUsingInvalidIntrospect = false;
```
Comment (ayoho) [inline dev/com.ibm.ws.security.fat.common.backchannelLogout/fat/src/com/ibm/ws/security/backchannelLogout/fat/utils/AfterLogoutStates.java:68]: We'll talk with Teddy next week about whether this flow is actually needed.

File: dev/com.ibm.ws.security.fat.common.backchannelLogout/publish/files/serversettings/oidcProvider_configTests.xml [hunk 2]
```diff
@@ -792,5 +805,122 @@
...
+			</client>
+		</localStore>
+	</oauthProvider>			
+
+	<openidConnectProvider
+		id="OidcConfigSample_appPasswords"
+		signatureAlgorithm="RS256"
+		keyAliasName="rs256"
+		keyStoreRef="key_allSigAlg"
+		tokenEndpointAuthMethodsSupported="client_secret_post, client_secret_basic, none"
```
Comment (ayoho) [inline dev/com.ibm.ws.security.fat.common.backchannelLogout/publish/files/serversettings/oidcProvider_configTests.xml:118]: Any particular reason why you need `none` in this list?

File: dev/com.ibm.ws.security.fat.common.backchannelLogout/publish/files/serversettings/oidcProvider_configTests_notLocalStore.xml [hunk 3]
```diff
@@ -281,4 +282,61 @@
 		
 	</oauthProvider>			
 			
+	<openidConnectProvider
+		id="OidcConfigSample_userClientTokenLimit"
+		signatureAlgorithm="RS256"
+		keyAliasName="rs256"
+		keyStoreRef="key_allSigAlg"
+		oauthProviderRef="OAuthConfigSample_userClientTokenLimit" />
```
Comment (ayoho) [inline dev/com.ibm.ws.security.fat.common.backchannelLogout/publish/files/serversettings/oidcProvider_configTests_notLocalStore.xml]: Would it make sense to have the `<openidConnectProvider>` elements in their own server.xml snippet and then put the localStore and Mongo `<oauthProvider>` instances in their own separate XMLs? Then we don't have to duplicate the OIDC config and can load the appropriate localStore/Mongo config.

**Review by ayoho — DISMISSED** [review]

**Comment by ayoho** [comment]
Closing since this should be superseded by https://github.com/OpenLiberty/open-liberty/pull/26763.

**Pull request closed without merging**